src/mlpack/methods/linear_svm/linear_svm_impl.hpp

```diff
@@ -182,17 +323,30 @@ void LinearSVM<MatType>::Classify(
   }
 }
 
-template <typename MatType>
-template <typename VecType>
-size_t LinearSVM<MatType>::Classify(const VecType& point) const
+template<typename ModelMatType>
+template<typename VecType>
+size_t LinearSVM<ModelMatType>::Classify(const VecType& point) const
 {
   arma::Row<size_t> label(1);
   Classify(point, label);
   return size_t(label(0));
 }
 
-template <typename MatType>
-double LinearSVM<MatType>::ComputeAccuracy(
+template<typename ModelMatType>
+template<typename VecType>
+void LinearSVM<ModelMatType>::Classify(
+    const VecType& point,
+    size_t& label,
+    typename LinearSVM<ModelMatType>::DenseColType& probabilities) const
+{
+  arma::Row<size_t> labelRow(1);
+  Classify(point, labelRow, probabilities);
+  label = labelRow[0];
+}
+
+template<typename ModelMatType>
+template<typename MatType>
+double LinearSVM<ModelMatType>::ComputeAccuracy(
```

Comment:
Can we make the return value `ElemType` please?

Reply:
(See also [this related comment/response](https://github.com/mlpack/mlpack/pull/3572#discussion_r1431934932).)  I don't mind returning the accuracy as a `double`; it won't have any real computational implications, and makes it so that when the user changes `ElemType`, there are fewer methods whose signatures change.